### ISSUE TITLE
Missing data due to router in Frog 1 being turned off

### ISSUE THREAD

**Issue by carlosparadis** [erdl/legacy-scrape-util#26]
According to @eileenpeppard the router of Frog 1 was turned off on August 10, and this was addressed last week. However, we have some missing data issues now. Quoting @eileenpeppard 

> Webctrl shows the data just fine, but scrape-util is not getting the data. Last date in database for frog 1 is 8/10 at 10:10.

**Comment by carlosparadis** [comment]
@ryantanaka confirmed that egauge data is up to date and the scrape-util state timestamp for egauge works, however something weird is going on with the webctrl. Some rows are being inserted, but the state timestamp is not updating, despite the rows being inserted one by one. @ryantanaka also confirmed that the webctrl data is likely fully downloaded correctly by scrape-util, but the insertion doesn't appear to be working. 

@ryantanaka will follow-up by setting some prints to check.

**Comment by ryantanaka** [comment]
The problem was that from 8-10 till 8-17, data from the SOURCE was missing. We checked by querying the webctrl api directly. For some reason scrape-util state timestamps were lagging behind at 8-3 (oddly since we were missing data from 8-10). 

I've updated the lagging timestamps for the frog1 webctrl sensors to be set at 08-18-2018 00:00:00 and this appears to have set things back in motion. Scrape util pulled and inserted all the data from then until now into the database. @eileenpeppard will check tableau/her database software to confirm this and get back to me so we can close up this issue. 

Also, data for the sensor associated with purpose_id 34 was being updated (the gap in data still exists) and we do not now why this happened.
